### ISSUE TITLE
Usage Clarification

### ISSUE THREAD

**Issue by jjzmajic** [aria2/aria2#1566]
Is there a way to download multiple files in parallel, each from multiple URLs? This works for a single url and multiple files:
```txt
http://aria2.net/files/stable/aria2-1.9.5/aria2-1.9.5.tar.bz2
  dir=/file/old
  out=aria2.old.tar.bz2
http://aria2.net/files/stable/aria2-1.10.0/aria2-1.10.0.tar.bz2
  out=aria2.new.tar.bz2
```

My question is:
1. Does this actually download the files in parallel or is it tantamount to `--force-sequential`?
2. Is there a way to specify multiple URLs for each file?

**Comment by VaslD** [comment]
### Documentation

http://aria2.github.io/manual/en/html/aria2c.html#input-file

### TL;DR:

Your example is an input file (or saved session file). This file can be created manually or by other apps; Aria2 also uses this file internally to track downloads between restarts. Aria2 reads this file as if each line is passed as a command line argument.

So to answer your question:

1. Without other interference, all downloads start in parallel as if you added them all at once. Sequential is not by default but can be configured globally either by adding command line arguments or adding options in config file. So the actual/final behavior may depend on other factors not shown in your input file example.

2. To use multiple URIs/URLs for a single output, specify multiple links for the same source in a single line separated by a tab character (`\t`, but not escaped like this). Then add your regular `dir=`, `out=`, and other options starting at the next line.

**Comment by jjzmajic** [comment]
Thanks @VaslD! I wasn't completely sure about the implications using a control file on the options passed to the program. This clears it up.
